mbtools/utils.py

```diff
@@ -3,14 +3,45 @@
 
 
 def parse_backup_activities(mbz_dir):
-    """Given a string with path to an extracted moodle backup directory return
+    """
+    Given a string with path to an extracted moodle backup directory return
     model objects for course activities
     """
     mbz_path = Path(mbz_dir).resolve(strict=True)
     moodle_backup = models.MoodleBackup(mbz_path)
     return moodle_backup.activities()
 
 
+def parse_question_bank_for_html(mbz_dir):
+    """
+    Given a string with path to a question_bank directory from an extracted
+    moodle backup, return model for a question bank
+    """
+
+    mbz_path = Path(mbz_dir).resolve(strict=True)
+    question_objects = models.MoodleQuestionBank(mbz_path).questions()
+    html = []
+    for question in question_objects:
+        for item in question.get_all_html_elements():
+            html.append(item.text)
+    return html
```

Comment:
I suspect we don't want to return strings here but instead `MoodleHtmlElement` objects so we can call things like `find_references_containing()` in the validator, etc.